chart/openfaas/templates/ingress-operator-crd.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.ingressOperator.create }}
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: functioningresses.openfaas.com
```

Comment:
I haven't tested the new CRD with the operator's Function CRD yet.. could you confirm that both work on a new cluster before I merge?

Reply:
I didn't understand what I should test.

Is this it?
- Create a new cluster
- Create a function with the `Function` CRD
- Create a `FunctionIngress` for that function

If not, could you give me some more information?

Cheers!

Reply:
Yep basically that please.